tooling/autodispose-error-prone-checker/src/test/java/com/uber/autodispose/error/prone/checker/MissingAutoDisposeErrorCheckerTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.autodispose.error.prone.checker;
+
+import com.google.errorprone.CompilationTestHelper;
+import java.util.Arrays;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+@SuppressWarnings("CheckTestExtendsBaseClass")
```

Comment:
Remove this, copy-pasta from internal

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/47c35ce452fe3965ac92bf3240679f65dc5ffddc

Reply:
Cool. Please make this a separate commit in the future though, that commit was mostly unrelated changes